api/test/e2enew/route_online_debug/route_online_debug_suite_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 func TestRoute(t *testing.T) {
 	gomega.RegisterFailHandler(ginkgo.Fail)
-	ginkgo.RunSpecs(t, "route online debug suite")
```

Comment:
@liuxiran how about removing this line instead of commenting out, just like other changes. While I think we could merge it first. 😄

Reply:
> @liuxiran how about removing this line instead of commenting out, just like other changes. While I think we could merge it first.

yep, it should be removed or at least add a comment statement why it is commented out, It will be done with other code optimizations later, thanks